.travis.yml

```diff
@@ -89,8 +89,11 @@ install:
   - if [ "$DO_CHECKQA" = 1 ]; then travis_retry sudo luarocks install luacheck; fi
 
   # Install ldoc for building docs.
-  - travis_retry sudo luarocks install ldoc 1.4.4
-  - travis_retry sudo luarocks install lua-discount
+  - |
+    if [ "$BUILD_APIDOC" == "true" ]; then
+      travis_retry sudo luarocks install ldoc 1.4.4
+      travis_retry sudo luarocks install lua-discount
+    fi
```

Comment:
As Daniel already pointed out: Doesn't this make changes in the CMake files from earlier commits obsolete?